src/fenicsxconcrete/experimental_setup/cantilever_beam.py

```diff
@@ -27,44 +29,56 @@ def __init__(self, parameters: dict[str, pint.Quantity]):
 
         super().__init__(default_p)
 
-    def setup(self):
+    def setup(self) -> None:
         """defines the mesh for 2D or 3D"""
 
-        if self.p['dim'] == 2:
-            self.mesh = df.mesh.create_rectangle(comm=MPI.COMM_WORLD,
-                                                 points=[(0.0, 0.0),
-                                                         (self.p['length'], self.p['height'])],
-                                                 n=(self.p['num_elements_length'],
-                                                    self.p['num_elements_height']),
-                                                 cell_type=df.mesh.CellType.quadrilateral)
-        elif self.p['dim'] == 3:
-            self.mesh = df.mesh.create_box(comm=MPI.COMM_WORLD,
-                                           points=[(0.0, 0.0, 0.0),
-                                                   (self.p['length'], self.p['width'], self.p['height'])],
-                                           n=[self.p['num_elements_length'],
-                                              self.p['num_elements_width'],
-                                              self.p['num_elements_height']],
-                                           cell_type=df.mesh.CellType.hexahedron)
+        if self.p["dim"] == 2:
+            self.mesh = df.mesh.create_rectangle(
+                comm=MPI.COMM_WORLD,
+                points=[(0.0, 0.0), (self.p["length"], self.p["height"])],
+                n=(self.p["num_elements_length"], self.p["num_elements_height"]),
+                cell_type=df.mesh.CellType.quadrilateral,
+            )
+        elif self.p["dim"] == 3:
+            self.mesh = df.mesh.create_box(
+                comm=MPI.COMM_WORLD,
+                points=[
+                    (0.0, 0.0, 0.0),
+                    (self.p["length"], self.p["width"], self.p["height"]),
+                ],
+                n=[
+                    self.p["num_elements_length"],
+                    self.p["num_elements_width"],
+                    self.p["num_elements_height"],
+                ],
+                cell_type=df.mesh.CellType.hexahedron,
+            )
```

Comment:
this change has nothing to do with typing ...
but I agree that we should employ some form of formatting via e.g. `flake8` and `black` for this project.

Reply:
Yes, I'm sorry. I use black formating as a standard, so it changed. 
I add a new issue for formatting. My suggestion would be including taht in teh git workflow